03week/airplane/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE>
+<html>
+<link href="css/style.css" rel="stylesheet" type="text/css"/>
+
+<header class="topnav">
+  <img class="logo" src="img/logo.png"/>
+  <nav>
+    <ul>
+       <li> <img src="img/login-icon.png"/> <a href="#">Register/Login</a> </li>
+       <li> <img src="img/schedule-icon.png"/> <a href="#">Flight Schedule</a></li>
+       <li> <img src="img/deal-icon.png"/> <a href="#">Travel Deals</a> </li>
+       <li> <img src="img/home-icon.png"/> <a href="#">Home</a> </li>
+    </ul>
+  </nav>
+</header>
+
+<main>
+<body>
+
+<div class="booking-container">
+    <nav class="booking-options">
+      <div class="active"><img src="img/plane-icon.png"/></div>
+      <div ><img src="img/car-icon.png"/></div>
+      <div  ><img src="img/hotel-icon.png"/></div>
+    </nav>
+    <div class="form-text">
+       <input class="wheretogo" type="text" placeholder="Depart">
```

Comment:
I would recommend that instead of smashing all the words together use `-` to simulate a space.
Example - `class="where-to-go"`. Makes it easier to read.